frontend/src/views/Landing.tsx

```diff
@@ -1,27 +1,205 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import copy from 'copy-to-clipboard';
 import { PrimaryButtonLink, TextLink } from '../components/core/Link';
-import { MainHeaderText, LandingHeaderTitle } from '../components/core/Text';
+import { Image, ShadowImage } from '../components/core/Image';
+import {
+  MainHeaderText, LandingHeaderTitle, SecondaryHeaderText, LargeText,
+} from '../components/core/Text';
+import { ColumnContainer, RowContainer, Separator } from '../components/core/Container';
+import { CopyIndicator, CopyIndicatorContainer, InheritedCopyIcon } from '../components/special/CopyIndicator';
+import { InheritedTextButton } from '../components/core/Button';
+
+const CreateAccountButtonLink = styled(PrimaryButtonLink)`
+  margin: 10px 0;
+`;
+
+const HeroTextContainer = styled.div`
+  text-align: left;
+  
+  @media(max-width: 1000px) {
+    text-align: center;
+    margin-bottom: 40px;
+  }
+`;
+
+const HeroText = styled(LandingHeaderTitle)`
+  font-size: ${({ theme }) => theme.fontSize.xxxLarge};
+  margin: 5px 0;
+  line-height: 1.2;
+  
+  @media(max-width: 1450px) {
+    font-size: ${({ theme }) => theme.fontSize.xxLarge};
+  }
+`;
+
+const HeroSubtitleText = styled(MainHeaderText)`
+  @media(max-width: 1450px) {
+    font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+  }
+`;
+
+const BackgroundCircleRow = styled(RowContainer)`
+  background-image: url("/landing/background_circles.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 150px 30px;
+  margin-bottom: 50px;
+  
+  @media(max-width: 1000px) {
+    padding: 0;
+    background-image: none;
+  }
+`;
+
+const CalToActionColumn = styled(ColumnContainer)`
+  flex: 0 0 60%;
+
+  @media(max-width: 1000px) {
+    width: 70%;
+    margin: 0 auto;
+  }
+`;
 
 function LandingPage() {
+  const [copiedEmail, setCopiedEmail] = useState(false);
+
   return (
     <>
-      <LandingHeaderTitle>
-        CodeJoust
-      </LandingHeaderTitle>
-      <MainHeaderText>
-        Compete live against friends and classmates to solve coding challenges.
-      </MainHeaderText>
-      <PrimaryButtonLink
-        width="12rem"
-        height="3.25rem"
-        to="/game/create"
-      >
-        Create a room
-      </PrimaryButtonLink>
-      <br />
-      <TextLink to="/game/join">
-        Or join an existing room &#8594;
-      </TextLink>
+      <CopyIndicatorContainer copied={copiedEmail}>
+        <CopyIndicator onClick={() => setCopiedEmail(false)}>
+          Link copied!&nbsp;&nbsp;✕
+        </CopyIndicator>
+      </CopyIndicatorContainer>
+
+      <RowContainer>
+        <ColumnContainer width="40%">
+          <HeroTextContainer>
+            <HeroText>Group coding made fun</HeroText>
```

Comment:
Sounds good, I'll bunch all of these into one commit as suggested

For this one in particular, after making the change, I also increased the size of the hero text to 3.3rem instead of 3.2, that on the larger screensizes the text will split across two lines nicely (before only "fun" would be on the second line)

Reply:
Awesome sounds great, thanks!